### PR TITLE
Add tyme-3-time-tracker extension

### PR DESCRIPTION
**Pull request by melkstam**
## Description

This extension enables users of [Tyme 3](https://www.tyme-app.com/) to start and stop tracking tasks. It does so using [their AppleScript support](https://www.tyme-app.com/en/scripting/). 

It has two commands, one for searching for and starting to track a task and one for stopping to track the current task.

## Screencast

Here is a simple Loom

https://www.loom.com/share/bb27be80a30b44c395c7f64b825df825?sid=a12c71a5-9c45-492a-9a98-cd230dae90ab

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/tyme-3-time-tracker/src/start-task-tracking.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const { data: tasks, isLoading: tasksIsLoading } = useCachedPromise(() => getTasks(), [], {
+    keepPreviousData: true,
+  });
+
+  // Sort by frequency
+  const { data: sortedTasks, visitItem: visitTask, resetRanking: resetTaskRanking } = useFrecencySorting(tasks);
+
+  // Handle start tracking
+  const handleStartTracking = (task: Task) => {
+    closeMainWindow({ clearRootSearch: true });
```
Comment (pernielsentikaer) [inline extensions/tyme-3-time-tracker/src/start-task-tracking.tsx:22]: Maybe add a HUD that it started tracking?

File: extensions/tyme-3-time-tracker/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "categories": [
+    "Productivity",
+    "Applications"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "start-task-tracking",
+      "title": "Start Task",
+      "subtitle": "Start Tracking a Tyme Task",
```
Comment (pernielsentikaer) [inline extensions/tyme-3-time-tracker/package.json:17]:
```suggestion
      "subtitle": "Tyme 3 Time Tracker",
```

File: extensions/tyme-3-time-tracker/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "description": "This command will let you search for any uncompleted task or subtask in Tyme 3 and start the time tracker for that task.",
+      "mode": "view"
+    },
+    {
+      "name": "stop-task-tracking",
+      "title": "Stop Task",
+      "subtitle": "Stop Tracking the Current Tyme Task",
```
Comment (pernielsentikaer) [inline extensions/tyme-3-time-tracker/package.json:24]:
```suggestion
      "subtitle": "Tyme 3 Time Tracker",
```

File: extensions/tyme-3-time-tracker/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Tyme 3 Time Tracker
+
+Interact with Tyme 3 to track your time.
+
+![Extension preview](./files/tyme-3-time-tracker-1.png)
```
Comment (pernielsentikaer) [inline extensions/tyme-3-time-tracker/README.md:5]:
```suggestion
![Extension preview](./metadata/tyme-3-time-tracker-1.png)
```
You delete the one from files

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/tyme-3-time-tracker/src/start-task-tracking.tsx (hunk 1 above)
Comment (melkstam) [reply extensions/tyme-3-time-tracker/src/start-task-tracking.tsx:22]: Good idea!

**Comment by melkstam** [comment]
Hi @pernielsentikaer! 👋

Thanks for your review! I've now addressed your comments so please re-review when you have the time.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**